src/reducers/versions.tsx

```diff
@@ -610,6 +612,29 @@ export const fetchDiff = ({
   };
 };
 
+type UpdateSelectedPathParams = {
```

Comment:
This should probably be renamed to `ViewVersionFileParams`.

Reply:
Ha! :facepalm: